source/retryable-writes/tests/insertOne-errorLabels.yml

```diff
@@ -0,0 +1,80 @@
+runOn:
+    -
+        minServerVersion: "4.3.0"
+        topology: ["replicaset", "sharded"]
+
+data: []
+
+tests:
+    -
+        description: "InsertOne succeeds after RetryableWriteError from server"
+        failPoint:
+            configureFailPoint: failCommand
+            mode: { times: 1 }
+            data:
+                failCommands: ["insert"]
+                errorLabels: ["RetryableWriteError"]
+                errorCode: 112 # WriteConflict, not normally a retryable error code
+            operation:
+                name: "insertOne"
+                arguments:
+                    document: { _id: 1, x: 11 }
+            outcome:
+                result:
+                    insertedId: 1
+                collection:
+                    data:
+                        - { _id: 1, x: 11 }
+    -
+        description: "InsertOne succeeds after a connection failure"
```

Comment:
This looks like it'd duplicated from `insertOne-serverErrors.yml`. Should this be deleted?